samples/web/content/testrtc/js/camresolutionstest.js

```diff
@@ -120,47 +206,119 @@ CamResolutionsTest.prototype = {
       }
     }
 
-    var avgEncodeMs = arrayAverage(googAvgEncodeTime);
-    var minEncodeMs = arrayMin(googAvgEncodeTime);
-    var maxEncodeMs = arrayMax(googAvgEncodeTime);
-    var avgFPSInput = arrayAverage(googAvgFrameRateInput);
-    var minFPSInput = arrayMin(googAvgFrameRateInput);
-    var maxFPSInput = arrayMax(googAvgFrameRateInput);
-    var avgFPSSent = arrayAverage(googAvgFrameRateSent);
-    var minFPSSent = arrayMin(googAvgFrameRateSent);
-    var maxFPSSent = arrayMax(googAvgFrameRateSent);
-    report.traceEventInstant('video-stats', {width: currentRes[0],
-                                             height: currentRes[1],
-                                             minEncodeMs: minEncodeMs,
-                                             maxEncodeMs: maxEncodeMs,
-                                             avgEncodeMs: avgEncodeMs,
-                                             minFPSInput: minFPSInput,
-                                             maxFPSInput: maxFPSInput,
-                                             avgFPSInput: avgFPSInput,
-                                             minFPSSent: minFPSSent,
-                                             maxFPSSent: maxFPSSent,
-                                             avgFPSSent: avgFPSSent});
-
     if (googAvgEncodeTime.length === 0) {
       reportError('No stats collected. Check your camera.');
     } else {
-      reportInfo('Encode time (ms): ' + minEncodeMs + ' min / ' + avgEncodeMs +
-                 ' avg / ' + maxEncodeMs + ' max');
-      reportInfo('Input FPS: ' + minFPSInput + ' min / ' + avgFPSInput +
-                 ' avg / ' + maxFPSInput + ' max');
-      reportInfo('Sent FPS: ' + minFPSSent + ' min / ' + avgFPSSent +
-                 ' avg / ' + maxFPSSent + ' max');
-      if (avgFPSSent < 5) {
-        reportError('Low average sent FPS: ' + avgFPSSent);
-      }
+      // TODO: Add a reportInfo() function with a table format to display
+      // values clearer.
+      statsReport.actualVideoWidth = this.video.videoWidth;
+      statsReport.actualVideoHeight = this.video.videoHeight;
+      statsReport.encodeSetupTime = this.timeForFirstFrameMs + ' (ms)';
```

Comment:
I don't like adding the (ms) and so on here... e.g. code further one can expect this to be an int and not a string.